bash/bash.sh

```diff
@@ -39,7 +39,13 @@ mkcd() {
 #jar opener with settings
 pjava() {
     if [ -e ./"$1" ]; then
-        java -Xmx650m -Xms650m -XX:+AlwaysPreTouch -XX:+DisableExplicitGC -XX:+UseG1GC -XX:+UnlockExperimentalVMOptions -XX:MaxGCPauseMillis=45 -XX:TargetSurvivorRatio=90 -XX:G1NewSizePercent=50 -XX:G1MaxNewSizePercent=80 -XX:InitiatingHeapOccupancyPercent=10 -XX:G1MixedGCLiveThresholdPercent=50 -XX:+AggressiveOpts -jar "$1"
+        java -Xmx650m -Xms650m -XX:+AlwaysPreTouch -XX:+DisableExplicitGC\
+            XX:+UseG1GC -XX:+UnlockExperimentalVMOptions\
+            XX:MaxGCPauseMillis=45 -XX:TargetSurvivorRatio=90\
+            XX:G1NewSizePercent=50 -XX:G1MaxNewSizePercent=80\
+            XX:InitiatingHeapOccupancyPercent=10\
+            XX:G1MixedGCLiveThresholdPercent=50\
+            XX:+AggressiveOpts -jar "$1"
```

Comment:
I believe this is far easier to read, rather than scrolling off-screen, moving away from other code you might want to look at.